src/current/v24.1/disaster-recovery-overview.md

```diff
@@ -0,0 +1,172 @@
+{{site.data.alerts.end}}
+
+## Resilience strategy
+
+As you evaluate CockroachDB's disaster recovery features, consider your organization's requirements for the amount of tolerable data loss and the acceptable length of time to recover.
+
+- Recovery Point Objective (RPO): The maximum amount of time that an organization can tolerate losing data.
+- Recovery Time Objective (RTO): The maximum length of time it should take to restore normal operations following an outage.
+
+For example, when you use backups:
+
+<img src="{{ 'images/v24.1/rpo-rto.png' | relative_url }}" alt="Simulating RPO and RTO. With RPO representing the tolerable data loss, and RTO representing the tolerable time to recovery." style="border:0px solid #eee;max-width:100%" />
+
+For a comparison of CockroachDB resiliency features, refer to the following sections:
+
+- [Choose a high availability strategy](#choose-a-high-availability-strategy)
+- [Choose a disaster recovery strategy](#choose-a-disaster-recovery-strategy)
+
+### Choose a high availability strategy
+
+CockroachDB uses synchronous, built-in replication to create and distribute copies of data, ensuring consistency across the data copies. The database can tolerate nodes going offline without service interruption whether in a single-region or [multi-region]({% link {{ page.version.version }}/multiregion-overview.md %}) cluster.
+
+<table class="comparison-chart">
+  <tr>
+    <th></th>
+    <th>Single-region replication</th>
+    <th>Multi-region replication</th>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RPO</b>
+    </td>
+    <td>0 seconds</td>
+    <td>0 seconds</a></td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RTO</b>
+    </td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Write latency</b>
+    </td>
+    <td>Region-local write latency<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+    <td>Cross-region write latency<br>p50 latency > 50ms  (for a multi-region cluster in us-east1, us-east-2, us-west-1)</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Recovery</b>
+    </td>
+    <td>Automatic</td>
+    <td>Automatic</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Hardware cost</b>
+    </td>
+    <td>1x<br>Networking between availability zones</td>
+    <td>1x<br>Networking between datacenters</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Minimum regions</b>
+    </td>
+    <td>1</td>
+    <td>3</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Fault tolerance</b>
+    </td>
+    <td>Zero RPO node, availability zone failures</td>
+    <td>Zero RPO node, availability zone failures</td>
```

Comment:
The multi-region one  should also be able to survive a region failure

Reply:
Done!